app/services/rights_description_builder.rb

```diff
@@ -7,36 +7,39 @@ def self.build(cocina)
 
   def initialize(cocina)
```

Comment:
I think this would be a lot easier to understand if this was separated into two classes, one for collections and one for DROAcess or AdminPolicyDefaultAccess. That would make for many fewer conditionals.

Reply:
Ok, collections in one class, items/APOs in the other?  Items/APOs are more alike.

Reply:
Yes, exactly.